README.md

```diff
@@ -112,36 +129,49 @@ Available filters:
 
 # Frequently Asked Questions
 
+## How can I get help?
+
+If your question is not answered on this page:
+
+* For help with the plugin: Please use the [support forum on wordpress.org](https://wordpress.org/support/plugin/vipps-recurring-payments-gateway-for-woocommerce/) or [submit an issue](https://github.com/vippsas/vipps-recurring-woocommerce/issues) on GitHub.
+* For help with Vipps: Please see the
+  [contact us](https://github.com/vippsas/vipps-developers/blob/master/contact.md)
+  page, and also the main
+  [Vipps GitHub page](https://github.com/vippsas).
+  The
+  [Vipps eCom FAQ](https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api-faq.md) may also be useful.
```

Comment:
Was adding a link to the ecom API instead of the recurring API FAQ on purpose?

Reply:
We need both. Fixed now, thanks.